convex_api/utils.py

```diff
@@ -117,8 +117,10 @@ def add_0x_prefix(text: Union[str, None]) -> Union[str, None]:
     :returns: The text with a `0x` appended to the front
 
     """
-    if text:
```

Comment:
`if text:` is testing for text == None and len(text) == 0
`if text is None` is only testing for == None not len(text) == 0